cunumeric/module.py

```diff
@@ -730,7 +764,7 @@ def linspace(
         # sequences with 0 items or 1 item with endpoint=True (i.e. div <= 0)
         # have an undefined step
         step = np.NaN
-        if delta.nim == 0:
+        if delta.ndim == 0:
```

Comment:
FYI @magnatelee this bug existed previously, mypy picked it up after I started adding more types to `array.py` for a later PR, but I figured better to get a fix in sooner.